app/assets/javascripts/covid19/src/data/Covid19InfoFromJsonRepository.ts

```diff
@@ -217,6 +215,36 @@ function getDetails(pdb: Data.Pdb): Maybe<Details> {
     };
 }
 
+function getPdbValidations(pdb: Data.Pdb): PdbValidation[] {
+    return pdb.refModels
+        ? pdb.refModels?.map(refModel => {
+              switch (refModel.method) {
+                  case "PDB-Redo":
+                      return {
+                          type: "pdbRedo",
+                          badgeColor: "w3-orange",
+                          externalLink: refModel.externalLink,
+                          queryLink: refModel.queryLink,
+                      };
+                  case "Isolde":
+                      return {
+                          type: "isolde",
+                          badgeColor: "w3-cyan",
+                          queryLink: refModel.queryLink,
+                      };
+                  case "Refmac":
+                      return {
+                          type: "refmac",
+                          badgeColor: "w3-blue",
+                          queryLink: refModel.queryLink,
+                      };
+                  default:
```

Comment:
TS nows the switch on `refModel.method`is exhaustive, this shouldn't be necessary.